### DISCUSSION TITLE
Purple Launcher PlayNC

### DISCUSSION THREAD

**Discussion by aeron3950** [moraroy/NonSteamLaunchers-On-Steam-Deck#452] [Ideas]
i hope you add support for Purple Launcher PlayNC someday

**Comment by moraroy** [comment]
@aeron3950 can you send the link?

**Reply by aeron3950** [reply]
https://purple.plaync.com/?locale=en-US here you go sir

**Reply by moraroy** [reply]
@aeron3950 it installs fine, but i cant get it to run, not sure why
